### PR TITLE
add infinite scroll

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
ACK !

File: src/providers/wallet/wallet.ts [hunk 1]
```diff
@@ -368,18 +368,22 @@ export class WalletProvider {
 
   public getTxHistory(wallet: any, opts: any) {
     return new Promise((resolve, reject) => {
-      opts = opts ? opts : {};
+      opts = opts || {};
```
Comment (Gamboster) [inline src/providers/wallet/wallet.ts]: Why is this change necessary?

Comment (JDonadio) [reply src/providers/wallet/wallet.ts]: Just habit 😂
